sway/desktop/output.c

```diff
@@ -224,11 +224,146 @@ static void render_view(struct sway_view *view, struct sway_output *output) {
 }
 
 /**
- * Render decorations for a view with "border normal".
+ * Select a color to use to render a border based on focus.
+ */
+static struct border_colors *select_focus_color(char *name) {
```

Comment:
Could you remove this function and just refer to `&config->border_colors.foo` directly as needed?